src/pages/about.js

```diff
@@ -1,66 +1,118 @@
+            />
+          </div>
+        </div>
+      </section>
 
-        <p className="text-left leading-loose mt-4">
-          Examples of unacceptable behavior by participants include:
-        </p>
+      <section className="FCC-divider">
+        <h2>
+          <span>LEADERSHIP</span>
+        </h2>
+      </section>
 
-        <ul className="text-left">
-          <li>
-            The use of sexualized language or imagery and unwelcome sexual
-            attention or advances
-          </li>
-          <li>
-            Trolling, insulting/derogatory comments, and personal or political
-            attacks
-          </li>
-          <li>Public or private harassment</li>
-          <li>
-            Publishing others&apos; private information, such as a physical or
-            electronic address, without explicit permission
-          </li>
-          <li>
-            Other conduct which could reasonably be considered inappropriate in
-            a professional setting
-          </li>
-        </ul>
+      <section className="main-section section-content-2">
+        <div className="main-section-content flex flex-col  md:justify-around ">
+          <div className="flex flex-col  justify-around items-center md:flex-row">
+            <Img
+              className="rounded-full w-64 m-3 md:w-1/4"
+              fluid={props.data.avatarSethAlexanderImage.childImageSharp.fluid}
+              alt="Profile picture of Seth Alexander"
+            />
+            <p className="text-xl md:w-3/4">
+              <b>Seth Alexander</b>, freeCodeCamp Nashville Organizer, Software
+              Engineer, Hard Problem Solver, TypeScript Lover, freeCodeCamp
+              believer. I help facilitate learning and foster community in any
+              way I can. I host the monthly meetups which can include code
+              challenges, lectures, and panels.
+            </p>
+          </div>
+          <div className="p-4 flex flex-col  justify-around items-center md:flex-row">
+            <Img
+              className="rounded-full w-64 m-3 md:w-1/4"
+              fluid={props.data.avatarAlexThomasImage.childImageSharp.fluid}
+              alt="Profile picture of Seth Alexander"
+            />
+            <p className="text-xl md:w-3/4">
+              <b>Alex Thomas</b>, freeCodeCamp Nashville Organizer, Full-Stack
+              JavaScript developer with experience building and deploying web
+              applications and APIs. I run the weekly Algorithms and Data
+              Structure meetup because I love helping students and junior
+              developers improve their professional skills and work on career
+              goals.
+            </p>
+          </div>
+        </div>
       </section>
     </Layout>
   );
 }
 
+export const query = graphql`
+  query {
+    avatarSethAlexanderImage: file(relativePath: { eq: "Seth-Alexander.jpg" }) {
```

Comment:
What's going on on this line? I'll run a global search on these keywords to see if I can't make sense of it.